docs/ruleset_functions/normalize_space_schedules.md

```diff
@@ -0,0 +1,43 @@
+
+## normalize_space_schedules
+
+Description: This function would determine a normalized schedule for a data element in space.  
+
+Inputs:
+  - **Space.data_element**: The space data element that needs to determine a normalized schedule, e.g. space.interior_lighting.
+
+Returns:
+- **space_normalized_schedule_array**: An array containing 8760 hourly values of a normalized schedule of the space data element.
+
+Function Call:
+
+- convert_schedule_to_hourly()
+
+Applicability:
+
+- space.interior_lighting
+- space.miscellaneous_equipment (To be developed)
+
+Logic:
+
+- For each interior lighting in space: `for interior_lighting in space.interior_lighting:`
+
+  - Get lighting power per area: `power_per_area = interior_lighting.power_per_area`
+
+  - Add lighting power per area to space total: `space_total_power_per_area += power_per_area`
+
+  - Get lighting schedule: `schedule = interior_lighting.lighting_multiplier_schedule`
+
+    - If schedule is hourly type, multiply hourly values with power per area and save to an 8760 hours array: `if schedule.schedule_sequence_type == "HOURLY": 8760_array.append(hourly_value * power_per_area for hourly_value in schedule.hourly_values)`
+
+    - Else, schedule is event type, convert schedule values to hourly: `else: converted_schedule_array = convert_schedule_to_hourly(schedule)`
+
+      - Multiply hourly values with power per area and save to an 8760 hours array: `8760_array.append(hourly_value * power_per_area for hourly_value in converted_schedule_array)`
+
```

Comment:
I am a littly confused by the nomenclature. After this step we should be getting an 8760 array of hourly_use_per_area (kWh/SF) for each interior lighting object. Is that what this is goind? If so, perhaps change the name of the resulting array to hourly_use_per_area?

Reply:
Updated.